ming_sim/urge_lever.py

```
@@ -0,0 +1,756 @@
+def _record_commitment_urge(
+    db: Any,
+    state: Any,
+    *,
+    commitment_ref: int,
+    stage_idx: int,
+    old_due: int,
+    new_due: int,
+    deadline_months: int,
+    reason: str,
+) -> None:
+    """committed 行作史源（不经 settle 批交；DELETE 仅清 pending）。"""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)